### PR TITLE
Added Language Malay

### PR DESCRIPTION
**Pull request by nasrulhazim**
I have added a Malay Language.

### CODE REVIEW

File: js/locales/bootstrap-datepicker.my.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Malay translation for bootstrap-datepicker
+ * Nasrul Hazim <http://github.com/nasrulhazim>
+ */
+$.fn.datepicker.dates['my'] = {
```
Comment (acrobat) [inline js/locales/bootstrap-datepicker.my.js]: Shouldn't "ms" be the language code? See: https://en.wikipedia.org/?title=Malay_language right column at ISO 639-1

Comment (nasrulhazim) [inline js/locales/bootstrap-datepicker.my.js]: Yerp, my mistake. will update and make another pull-request.

Comment (acrobat) [inline js/locales/bootstrap-datepicker.my.js]: Another PR is not necessary, just add another commit to this current branch!

**Comment by hebbet** [comment]
@nasrulhazim can you fix the build error?

File: js/locales/bootstrap-datepicker.ms.js [hunk 2]
```diff
@@ -9,6 +9,6 @@
 		daysMin: ["Ah", "Is", "Se", "Ra", "Kh", "Ju", "Sa"],
 		months: ["Januari", "Februari", "Mac", "April", "Mei", "Jun", "Julai", "Ogos", "September", "Oktober", "November", "Disember"],
 		monthsShort: ["Jan", "Feb", "Mar", "Apr", "Mei", "Jun", "Jul", "Ogo", "Sep", "Okt", "Nov", "Dis"],
-		today: "Hari Ini"
+		today: "Hari Ini",
+    clear: "Bersihkan"
```
Comment (acrobat) [inline js/locales/bootstrap-datepicker.ms.js]: please remove the space before the clear key, to keep alignment over all keys!

File: js/locales/bootstrap-datepicker.ms.js [hunk 3]
```diff
@@ -9,6 +9,6 @@
...
 	};
-}(jQuery));
```
Comment (acrobat) [inline js/locales/bootstrap-datepicker.ms.js]: Restore this line

**Comment by amin007** [comment]
Menarik, ada butang bersihkan :+1:

**Pull request closed without merging**